iphone/Classes/APIModule.m

```diff
@@ -71,17 +52,17 @@ - (id)transform:(id)arg
   return arg;
 }
 
-- (void)debug:(NSArray *)args
+- (void)debug:(id)args
```

Comment:
See my comment below. I believe the new API binding actually does allow you to use the actual types, but if any given method accepts more than one type (or we used to implicitly coerce it ourselves before) it would fail. My way to hack around that was to move to `id` and handle the cases myself if we accept differing types.